plugins/bufbuild/connect-web/v0.3.2/buf.plugin.yaml

```diff
@@ -0,0 +1,19 @@
+version: v1
+name: buf.build/bufbuild/connect-web
+plugin_version: v0.3.2
+source_url: https://github.com/bufbuild/connect-web
+description: Generates client stubs for calling Connect and gRPC-Web services from a web browser.
+deps:
+  - plugin: buf.build/bufbuild/es:v0.3.0
```

Comment:
Updated manually.

Reply:
Also, we need to be careful with our crude operations to update versions. If there is an overlap between the incoming version and a dependency version we'll end up bumping it.

We might want to consider excluding dependency bumps when doing a search/replace.